src/main/scala/com/workday/warp/persistence/influxdb/InfluxDBClient.scala

```diff
@@ -157,9 +157,20 @@ trait InfluxDBClient extends StackTraceFilter with CorePersistenceAware {
    * @return true iff databaseName exists as a database in InfluxDB and we have a successful connection.
    */
   def databaseExists(databaseName: String): Try[Boolean] = {
-    InfluxDBClient.client match {
+    val showQuery: Query = new Query("SHOW DATABASES", databaseName)
+    InfluxDBClient.maybeClient match {
       case None => Failure(new WarpConfigurationException(InfluxDBClient.error))
-      case Some(client) => Try(client.describeDatabases.asScala.exists(_.equals(databaseName)))
+      case Some(client) => Try {
+        val results: Seq[QueryResult.Result] = client.query(showQuery).getResults.asScala
+        val databaseNames: Seq[String] = for {
+          res <- results
+          serie <- res.getSeries.asScala
```

Comment:
did you want this to be `series` or did you mean to have it as `serie`

Reply:
hah its kinda dumb, but intentional. i wanted to clarify that its a single series, a `serie`. `getSeries` returns a `List<Series>`, where a `Series` has these fields:
```java
    private Map<String, String> tags;
    private List<String> columns;
    private List<List<Object>> values;
```